duckduckgo/Android
# Feature/david/apptp/waitlist survey

Task/Issue URL: https://app.asana.com/0/1174433894299346/1201623667172762

### Description 

### Steps to test this PR
- In `SurveyService`, change the url for surveyAppTP to https://ddg-sandbox.s3.amazonaws.com/survey/apptp/survey-mobile.json-
- Make sure to run playDebug flavor

_Survey available_
- [x] Open app and navigate to a couple of websites
- [x] Close the app
- [x] Open the app again and verify that the CTA for a survey does not appear
- [x] Join the AppTP Waitlist
- [x] Close the appo
- [x] Open the app and verify that the CTA for a survey appears
- [x] Verify the survey is DuckDuckGo Privacy Browser Survey
